### DISCUSSION TITLE
Strange developer registration form?

### DISCUSSION THREAD

**Discussion by Kalo79** [amzn/selling-partner-api-models#3581] [General] [answered]
Hello, 
I'm trying to create a developer account but the registration form does not look right. (see the attached screenshots) Anyone with the same problem?
![Screenshot 2023-04-18 131344](https://user-images.githubusercontent.com/59641322/232751867-adefd4d2-b119-4c4e-bff4-04ad7d72d93e.png)
![Screenshot 2023-04-18 131417](https://user-images.githubusercontent.com/59641322/232752199-7a236f86-8481-4b50-a4fe-f1c53fb5b42f.png)

**Comment by sungolivia** [comment]
I would highly contact our developer support team if you are seeing this consistently!
https://developer.amazonservices.com/support

**Comment by Kalo79** [comment]
I did, but I have no reply yet. 🙁

**Reply by sungolivia** [reply]
@Kalo79 any update?

**Comment by Kalo79** [comment, marked as answer]
Yes. Sorry for the delay! The form was fixed and I was able to register at
the end of last week. And today I got my approval email, so everything is
OK now.
